openhtf/exe/test_state.py

```diff
@@ -79,6 +83,12 @@ def __init__(self, config, test, plugs, dut_id):
     self.logger = logging.getLogger(test.filename)
     self.logger.setLevel(logging.DEBUG)  # Let the handler do the filtering.
     self.logger.addHandler(logs.RecordHandler(self.record))
+    formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
This seems to break the logging paradigm we discussed by re-converging the two logging streams. Let's discuss in person when I get back next week.